apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -434,18 +439,76 @@ export const OrganizationWrapper: React.FC<WrapperProps> = ({
+                  <Box
+                    marginY={
+                      organizationPage.secondaryMenu &&
+                      secondaryNavList.length > 0
+                        ? 0
+                        : 3
+                    }
+                    className={styles.sidebarCardContainer}
+                  >
+                    {organizationPage.sidebarCards.map((card) => {
+                      if (card.__typename === 'SidebarCard') {
+                        return (
+                          <ProfileCard
+                            key={card.id}
+                            title={card.title}
+                            description={card.contentString}
+                            link={card.link}
+                            image={
+                              card.image?.url ||
+                              'https://images.ctfassets.net/8k0h54kbe6bj/6jpT5mePCNk02nVrzVLzt2/6adca7c10cc927d25597452d59c2a873/bitmap.png'
+                            }
+                            size="small"
+                          />
+                        )
                       }
-                      size="small"
-                    />
-                  ))}
+
+                      if (
+                        card.__typename === 'ConnectedComponent' &&
+                        (card.type === 'LatestNewsCard' ||
+                          card['componentType'] === 'LatestNewsCard')
+                      ) {
+                        return (
+                          <LatestNewsCardConnectedComponent
+                            key={card.id}
+                            {...card.json}
+                          />
+                        )
+                      }
+
+                      if (card.__typename === 'MailingListSignupSlice') {
+                        return (
+                          <MailingListSignup
+                            id={'mailingListSignupForm-' + card.id}
+                            title={card.title}
+                            description={card.description}
+                            inputLabel={card.inputLabel}
+                            buttonText={card.buttonText}
+                            signupID={card.id}
+                            boxProps={{
+                              padding: 3,
+                              border: 'standard',
+                              borderRadius: 'standard',
+                            }}
+                            inputAndButtonContainerBoxProps={{
+                              display: 'flex',
+                              flexDirection: 'column',
+                              alignItems: 'flexStart',
+                            }}
+                            buttonBoxProps={{
+                              marginTop: 3,
+                            }}
+                            variant="white"
+                            namespace={namespace ?? (({} as unknown) as any)}
```

Comment:
Is there no better type that can be used?

Reply:
I changed the namespace type so this has been addressed :)